features/prompt_memes_b/challenge_promptmeme_posting_fills.feature

```diff
@@ -458,3 +459,41 @@ Feature: Prompt Meme Challenge
   Then I should see "In response to a prompt by myname4"
     And I should see "Fandom: Stargate Atlantis"
     And I should see "Anonymous" within ".byline"
+    And I should see a link "prompt"
+
+  Scenario: Fic links to the prompt it fulfils, for all users
+
+  Given I have Battle 12 prompt meme fully set up
+  When I am logged in as "myname1"
+  When I sign up for Battle 12 with combination B
+    And I am logged in as "myname4"
+    And I claim a prompt from "Battle 12"
+  When I fulfill my claim
+  When I reveal works for "Battle 12"
```

Comment:
Let's make all of these start with "And" and be indented under the initial "Given" step, since they're all sort of background/setup steps.